<launch>
  <node name="arm_node" pkg="zeus_arm" type="ArmNode.py" />
</launch>
